### PR TITLE
Add basic multistore in modules doc

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Copy old doc from here: http://doc.prestashop.com/display/PS16/Specifics+of+multistore+module+development

This doc might be outdated! Perhaps @matthieu-rolland and @kpodemski can confirm and help update it.

### CODE REVIEW

File: src/content/1.7/modules/concepts/multistore.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    )
+) {
+    $this->adv_imgname .= '-s'.$this->context->shop->getContextShopID();
+}
+$this->adv_img = Tools::getMediaServer($this->name)
+    . _MODULE_DIR_ . $this->name . '/' . $this->adv_imgname . '.' 
+    . Configuration::get('BLOCKADVERT_IMG_EXT');
+```
+
+## Creating database tables
```
Comment (matks) [inline src/content/1.7/modules/concepts/multistore.md:87]:
```suggestion
## Building database tables that support multistore
```

File: src/content/1.7/modules/concepts/multistore.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+There can be several scenarios:
+
+* The object/entity exists in some stores, but not in all of them: you should use an associative table, and handle it using `id_group_shop`.
+* The object/entity exists in all stores: use the regular tables, with a custom field. Handle it using `id_entity`, `id_group_shop` and the attributes.
+* The language must be different for every store: handle it using `id_shop` in the `_lang` table.
+
+### Object in some stores only
+
+The method to use is `Shop::addSqlRestriction($share = false, $alias = null)`.
```
Comment (matks) [inline src/content/1.7/modules/concepts/multistore.md:97]:
```suggestion
When building SQL queries that manage module data, the method to use is `Shop::addSqlRestriction($share = false, $alias = null)`.
```

**Comment by kpodemski** [comment]
👍 

There's a lot of advanced stuff related to `ObjectModel` and multi-store, especially some magic when you set `multishop_lang` to `true` in `$definition` but documenting it can take a lot of time, I'll make a note on my `to do` to get back to this topic when I will have more free time.

Also, I think it would be better to have an example based on some newer, not archived module but for now it's better to have these basics than nothing!

**Comment by matks** [comment]
@eternoendless 🆙  😉

**Review by kpodemski — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @eternoendless

**Pull request merged**